### PR TITLE
ttm classifier

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [ ] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

File: notebooks/data-sources/oc-github-repo/ceph_communication.ipynb [hunk 1]
```diff
@@ -0,0 +1,159 @@
+{
```
Comment (chauhankaranraj) [inline notebooks/data-sources/oc-github-repo/ceph_communication.ipynb]: I'm not sure if this notebook is required. Can't we just add the metric template notebook to the path in `github_PR_EDA.ipynb` nb, like this?
```python
metric_template_path = "../TestGrid/metrics"
if metric_template_path not in sys.path:
    sys.path.insert(1, metric_template_path)

from ipynb.fs.defs.metric_template import (    # noqa: E402
    CephCommunication,
    save_to_disk,
)
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
Just a comment about metric template stuff, otherwise /lgtm :)

File: notebooks/data-sources/oc-github-repo/ceph_communication.ipynb (hunk 1 above)
Comment (oindrillac) [reply notebooks/data-sources/oc-github-repo/ceph_communication.ipynb]: I wasnt sure if that's possible. Perfect, I will add this

Comment (oindrillac) [reply notebooks/data-sources/oc-github-repo/ceph_communication.ipynb]: @chauhankaranraj made the changes 🙂

**Review by chauhankaranraj — APPROVED** [review]

**Pull request merged**